.github/scripts/check-template-and-add-labels.ts

```diff
@@ -235,49 +246,79 @@ function extractReleaseVersionFromBugReportIssueBody(
   return version;
 }
 
-// This function adds the correct "regression-prod-x.y.z" label to the issue, and removes other ones
-async function addRegressionProdLabelToIssue(
+// This function adds the correct regression label to the issue, and removes other ones
+async function addRegressionLabelToIssue(
   octokit: InstanceType<typeof GitHub>,
-  releaseVersion: string,
   issue: Labelable,
 ): Promise<void> {
-  // Craft regression prod label to add
-  const regressionProdLabel: Label = {
-    name: `regression-prod-${releaseVersion}`,
-    color: '5319E7', // violet
-    description: `Regression bug that was found in production in release ${releaseVersion}`,
-  };
-
-  let regressionProdLabelFound: boolean = false;
-  const regressionProdLabelsToBeRemoved: {
+  // Extract regression stage from bug report issue body (if existing)
+  const regressionStage = extractRegressionStageFromBugReportIssueBody(
+    issue.body,
+  );
+
+  // Extract release version from bug report issue body (if existing)
+  const releaseVersion = extractReleaseVersionFromBugReportIssueBody(
+    issue.body,
+  );
+
+  // Craft regression label to add
+  let regressionLabel: Label;
```

Comment:
nit: maybe we can extract some logic and split it up into separate functions. Ie, we can convert this as a const and get the label from a separate function

`const regressionLabel = getRegressionLabel(regressionStage, releaseVersion);`

Reply:
Sure, it's now fixed in this [commit](https://github.com/MetaMask/metamask-extension/pull/25691/commits/0675b65ab5dea44f83c0c5ed785e62432e3306b5)